hack/build-base-images.sh

```diff
@@ -5,10 +5,29 @@
 STARTTIME=$(date +%s)
 source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
+HOST_ARCH=$(os::build::host_arch)
+
+DOCKERFILE="Dockerfile"
+
+if [[ "${HOST_ARCH}" != "amd64" ]]; then
+  DOCKERFILE+=".${HOST_ARCH}"
+fi
+
 # determine the correct tag prefix
 tag_prefix="${OS_IMAGE_PREFIX:-"openshift/origin"}"
 
+BASE_EXTRA_TAG=""
+if [[ "${HOST_ARCH}" == "amd64" ]]; then
+  BASE_EXTRA_TAG="${tag_prefix}-base"
+fi
+
 # Build the base image without the default image args
-OS_BUILD_IMAGE_ARGS="${OS_BUILD_IMAGE_BASE_ARGS-}" os::build::image "${OS_ROOT}/images/base" "${tag_prefix}-base"
+OS_BUILD_IMAGE_ARGS="${OS_BUILD_IMAGE_BASE_ARGS-}" os::build::image "${OS_ROOT}/images/base" "${tag_prefix}-base-${HOST_ARCH}" ${DOCKERFILE} ${BASE_EXTRA_TAG}
```

Comment:
While building docker images I'm planning to have <imagename>-<arch> to support multi arch like amd64, ppc64le etc.. to fallback to support existing model I'm tagging still tagging them as <imagename> only for amd64 without arch in it. 

e.g openshift/origin-base and openshift/origin-base-amd64 for amd64 arch
openshift/origin-base-ppc64le for ppc64le platform..